.github/workflows/ci.yml

```diff
@@ -244,18 +245,18 @@ jobs:
     needs: ['set_up_integration_infra']
     uses: './.github/workflows/snapshot-file-copy.yml'
     with:
-      workload_identity_provider: 'projects/921163060412/locations/global/workloadIdentityPools/github-pool-3b39/providers/github-provider'
-      service_account: 'pmap-3b39-ci-sa@pmap-ci.iam.gserviceaccount.com'
-      destination_prefix: 'pmap-ci/mapping/${{ needs.set_up_integration_infra.outputs.INTEG_TEST_OBJECT_PREFIX_OUTPUT }}'
+      workload_identity_provider: 'projects/gha-pmap-ci-i-f49bd9/locations/global/workloadIdentityPools/github-automation/providers/pmap-ci-i'
+      service_account: 'github-automation-bot@gha-pmap-ci-i-f49bd9.iam.gserviceaccount.com'
```

Comment:
Should we use env vars?

Reply:
I tried it. It will throw error https://github.com/abcxyz/pmap/actions/runs/6162982934/workflow

Reply:
I think resuable workflow doesn't support it. See https://github.com/orgs/community/discussions/26671